# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/xf86-video-rendition/hotfix-moduledir.patch
# Copyright (C) 2026 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

diff --git a/src/vboard.c b/src/vboard.c
index c43dd99..b952cde 100644
--- a/src/vboard.c
+++ b/src/vboard.c
@@ -23,7 +23,7 @@
 #include "cscode.h"
 
 /* Global imported during compile-time */
-static char MICROCODE_DIR [PATH_MAX] = MODULEDIR;
+static char MICROCODE_DIR [PATH_MAX] = MODULEDIR "/";
 
 /*
  * local function prototypes
